.circleci/Gemfile.lock

```diff
@@ -0,0 +1,187 @@
+GIT
+  remote: https://github.com/aws-amplify/amplify-ci-support
+  revision: bea40e79f1c871f1fb5ceb86aeb6b1f2a284ed87
+  branch: rjuliano/android-deploy
```

Comment:
Using the branch for the PR I have against the amplify-ci-support repo. That will hopefully give us a chance to test things out